_includes/matomo.html

```diff
@@ -0,0 +1,15 @@
+<!-- Matomo -->
+<script type="text/javascript">
+  var _paq = window._paq || [];
+  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="https://analytics.llnl.gov/";
+    _paq.push(['setTrackerUrl', u+'matomo.php']);
+    _paq.push(['setSiteId', '378']);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'matomo.js'; s.parentNode.insertBefore(g,s);
```

Comment:
Not a JS programmer. But does this need to be a single line (readability?)

Reply:
To be honest, I am not sure. This is verbatim what matomo produces and says to add to your site.  I was kind of viewing this the same way as vendoring a dependency (i.e., try to minimize changes from upstream). I anticipate that going forward, we will just keep copying-and-pasting whatever the analytics page says to include (if the code ever changes).